### PR TITLE
subject and some description specs

### PR DESCRIPTION
**Pull request by 0xF013**
https://community.openproject.org/work_packages/20184
https://community.openproject.org/work_packages/20185

for reviewers:
- [x] check the query renaming
- [x] check right click on a wp in the list (I am no longer sure which is the desired behaviour for a single wp being rightclicked)

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://0xf013-details-pane-feature-specs.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://0xf013-details-pane-feature-specs.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: spec/features/work_packages/details/inplace_editor/description_editor_spec.rb [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      expect(field.input_element).to be_visible
+      expect(field.input_element.tag_name).to eq 'textarea'
+    end
+    it 'renders formatting buttons'
+    it 'renders a preview button'
+    it 'prevents page navigation in edit mode'
+    it 'has a correct value for the textarea'
+    it 'displays the new HTML after save'
+  end
+end
```
Comment (floriank) [inline spec/features/work_packages/details/inplace_editor/description_editor_spec.rb]: what the hound said :+1:

Comment (0xF013) [inline spec/features/work_packages/details/inplace_editor/description_editor_spec.rb]: @floriank I don't know what's wrong here, I am pushing it with a newline

Comment (floriank) [inline spec/features/work_packages/details/inplace_editor/description_editor_spec.rb]: @houndci is dumb for some of the changes. nvm.

**Pull request merged**